### DISCUSSION TITLE
Laravel Domain Expert

### DISCUSSION THREAD

**Discussion by salehhashemi1992** [HassanZahirnia/laravel-package-ocean#79] [Package Suggestions]
Laravel Domain Expert is a package that helps you create and manage domain-driven directory structures in your Laravel application. It automates the process of setting up a new domain with the necessary folders, a controller, and a simple route file with a route group and a domain prefix.

https://github.com/salehhashemi1992/laravel-domain-expert
https://packagist.org/packages/salehhashemi/laravel-domain-expert

**Comment by salehhashemi1992** [comment]
Added AI features to the cmd

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!
